### PR TITLE
[#6964] deps: updates after hibernation

### PR DESCRIPTION
**Pull request by phillimorland**
install normally
still to do: 

- [x] check and fix the npm vulnerabilities if possible
- [x] investigate tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,64 +31,68 @@
...
     "eslint-plugin-react": "7.30.1",
     "eslint-plugin-react-native": "4.0.0",
     "eslint-plugin-react-native-a11y": "3.2.1",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-simple-import-sort": "7.0.0",
     "husky": "8.0.1",
     "jest": "28.1.2",
     "jest-expo": "45.0.1",
     "lint-staged": "13.0.3",
     "patch-package": "6.4.7",
```
Comment (khamui) [inline package.json:86]: maybe we can remove this package and the file in `patches/...` ?

File: package.json [hunk 2]
```diff
@@ -31,64 +31,68 @@
...
-    "react-native-screens": "3.11.1",
-    "react-native-web": "0.17.7",
-    "sentry-expo": "4.2.0",
+    "react-native-gesture-handler": "2.8.0",
+    "react-native-reanimated": "2.12.0",
+    "react-native-render-html": "6.3.4",
+    "react-native-safe-area-context": "4.4.1",
+    "react-native-screens": "3.18.0",
+    "react-native-web": "0.18.9",
+    "sentry-expo": "5.0.3",
```
Comment (khamui) [inline package.json:65]: any reason not to upgrade to v 6.0.0? After installing it "warns" me with:

`sentry-expo@5.0.3 - expected version: ~6.0.0`

File: package.json [hunk 3]
```diff
@@ -31,64 +31,68 @@
...
-    "expo-status-bar": "1.3.0",
-    "expo-updates": "0.13.3",
+    "expo": "47.0.13",
+    "expo-application": "5.0.1",
+    "expo-constants": "14.0.2",
+    "expo-device": "5.0.0",
+    "expo-font": "11.0.1",
+    "expo-image-picker": "14.0.2",
+    "expo-linear-gradient": "12.0.1",
+    "expo-linking": "3.1.0",
```
Comment (khamui) [inline package.json:48]: after installing I get the following warning. Does it work with that version?

`expo-linking@3.1.0 - expected version: ~3.3.0`

File: package.json [hunk 4]
```diff
@@ -31,64 +31,68 @@
...
-    "react-test-renderer": "17.0.2",
+    "react-test-renderer": "18.1.0",
     "webpack-dev-server": "4.9.3"
   },
   "overrides": {
     "eslint-plugin-react-native-a11y": {
       "eslint": "$eslint"
+    },
+    "expo": {
+      "json5": "2.2.3"
```
Comment (khamui) [inline package.json:95]: does this override a "sub-dependency" ? just out of curiosity

**Review by khamui — COMMENTED** [review]
Really cool! working great, just some warnings from expo and maybe we can remove the patch

File: package.json (hunk 1 above)
Comment (phillimorland) [reply package.json:86]: apparently yes :)

Comment (khamui) [reply package.json:86]: @phillimorland let's remove it then?

Comment (phillimorland) [reply package.json:86]: sorry, didn't push it

Comment (phillimorland) [reply package.json:85]: @khamui there now

Comment (phillimorland) [reply package.json:86]: issue: https://github.com/liqd/zt-app/issues/411

**Comment by khamui** [comment]
yeah!

**Pull request merged**